Makefile

```diff
@@ -261,3 +261,12 @@ update-bundles: ## Push policy bundles to quay.io and generate infra-deployments
 	@hack/update-bundles.sh
 
 #--------------------------------------------------------------------
+
+##@ Acceptance Tests
+
+.PHONY: acceptance
+acceptance: ## Run acceptance tests
+	@go build -o acceptance/bin/ec github.com/enterprise-contract/ec-cli
```

Comment:
We could be using `go run` from the acceptance test instead? One executing the test from an IDE could end up running unintended version of ec (and I just saw we're doing this from in ec-cli, though we need to, to instrument for code coverage)

Reply:
This is a bit tricky. We can get the ec command like this:

```go
import "github.com/enterprise-contract/ec-cli/cmd"
...
func validateInputWithPolicyConfig(ctx context.Context) (context.Context, error) {
	...
	ec := cmd.RootCmd

	ec.SetContext(context.Background())
	ec.SetArgs([]string{
		"validate",
		"input",
		"--file",
		ts.inputFileName,
		"--policy",
		ts.configFileName,
		"--strict=false",
	})

	var stdout bytes.Buffer
	var stderr bytes.Buffer
	ec.SetOut(&stdout)
	ec.SetErr(&stderr)
	...
```

But this means that all tests are sharing a single instance of the command. This will be a problem as soon as we have more than one scenario :disappointed: 

We could also use `NewRootCmd()` from `/cmd/root` but that creates a command with no sub-commands. We have to call call `.AddCommand(...)` to add the command structure. However! That doesn't work because the function that creates the validate input command is not exported :cry: I'll play around with it. Maybe it makes sense to export this function.

Reply:
I was trying to use the ec cli as a go dependency. It is not, at least currently, meant to be used this way. I took a different approach in: https://github.com/enterprise-contract/ec-policies/pull/867